client/src/app/signin/Form.test.tsx

```diff
@@ -0,0 +1,86 @@
+import { render, fireEvent, waitFor } from '@testing-library/react';
+import '@/__mocks__/next/router';
+import strings from '@/locales/strings.json';
+import Form from './Form';
+import * as nextAuthReact from 'next-auth/react';
+
+jest.mock('next-auth/react');
+const nextAuthReactMocked = nextAuthReact as jest.Mocked<typeof nextAuthReact>;
+
+describe('Form Component', () => {
+  it('should render without crashing', () => {
+    render(<Form />);
+  });
+
+  it('should show an alert when form is submitted with invalid data', async () => {
+    const { getByText, queryByText } = render(<Form />);
```

Comment:
Tengo entendido que no se usa mas deconstruir el objeto de render sino que luego de usar render podes llamar a screen.getByText etc.
Screen viene de la misma libreria que render

Reply:
dale voy a probar con screen

Reply:
Sobre esto creo que estaria bueno ponerse de acuerdo ya que en el codigo actual se esta usando de esta forma.
@marcegodoy @Mr-T-Smyth que les parece mejor dejar como esta o cambiar a `screen y userEvent` ? En caso de querer cambiarlo podemos crear un ticket para cambiarlo en el resto de los test

Reply:
Hay una diferencia práctica en esto, osea userEvent simula el touch de usuario lo que si llama al flujo de validación y todo lo que uno marca en front, fireEvent solo llama al evento exacto que le declaras. Y por el lado de screen, si usas el ítem deconstruido tenés que tener en cuenta de mantener actualizado el objeto y esto se hace solo con screen

Reply:
Dale lo modifico acá y creamos otro ticket para actualizar el resto de los test les parece ?

Reply:
Si estoy de acuerdo con @Mr-T-Smyth 
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

Reply:
@santimintegui No hay muchos test mas, te parece necesario crear otro ticket? capaz es un toque

Reply:
Dale lo hago acá